holidays/countries/bangladesh.py

```
@@ -12,34 +12,69 @@
     weekend = {FRI, SAT}
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, islamic_show_estimated: bool = True, **kwargs):
         InternationalHolidays.__init__(self)
+        ChristianHolidays.__init__(self)
+        IslamicHolidays.__init__(
+            self, cls=BangladeshIslamicHolidays, show_estimated=islamic_show_estimated
+        )
         super().__init__(*args, **kwargs)
 
+    def _add_mid_shaban_day(self, name):
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `_add_mid_shaban_day`

(ANN202)